packages/graphback-runtime-knex/src/KnexDBDataProvider.ts

```diff
@@ -1,7 +1,8 @@
 import { buildModelTableMap, getDatabaseArguments, ModelTableMap } from '@graphback/core';
 import { GraphQLObjectType } from 'graphql';
 import * as Knex from 'knex';
-import { GraphbackDataProvider, GraphbackPage, NoDataError, AdvancedFilter } from '@graphback/runtime';
+import { GraphbackDataProvider, GraphbackPage, NoDataError, AdvancedFilter } from '../../graphback-runtime/src';
```

Comment:
Please correct me if I'm wrong but shouldn't this stay '@graphback/runtime'?

Reply:
Good catch!